yarn.lock

```diff
@@ -30715,7 +30712,7 @@ node-pty@beta:
   languageName: node
   linkType: hard
 
-"vm2@npm:^3.9.19, vm2@npm:^3.9.3":
+"vm2@npm:^3.9.3":
```

Comment:
I was expecting this to be gone.
Could you `yarn why vm2 --recursive` to see where this is still coming from ?

Reply:
Looks like.
```

$ yarn why vm2
└─ degenerator@npm:3.0.1
   └─ vm2@npm:3.9.19 (via npm:^3.9.3)

$ yarn why degenerator
└─ pac-resolver@npm:5.0.0
   └─ degenerator@npm:3.0.1 (via npm:^3.0.1)

$ yarn why pac-resolver
└─ pac-proxy-agent@npm:5.0.0
   └─ pac-resolver@npm:5.0.0 (via npm:^5.0.0)

$ yarn why pac-proxy-agent
└─ proxy-agent@npm:5.0.0
   └─ pac-proxy-agent@npm:5.0.0 (via npm:^5.0.0)


$ yarn why proxy-agent
├─ @aws-amplify/amplify-category-notifications@workspace:packages/amplify-category-notifications
│  └─ proxy-agent@npm:5.0.0 (via npm:^5.0.0)
│
├─ @aws-amplify/amplify-cli-core@workspace:packages/amplify-cli-core
│  └─ proxy-agent@npm:5.0.0 (via npm:^5.0.0)
│
└─ @aws-amplify/amplify-provider-awscloudformation@workspace:packages/amplify-provider-awscloudformation
   └─ proxy-agent@npm:5.0.0 (via npm:^5.0.0)
 ```

Reply:
https://github.com/TooTallNate/proxy-agents/releases/tag/pac-resolver%407.0.0